source/default/_patterns/01-atoms/svg/_svg.twig

```diff
@@ -3,29 +3,43 @@
  * @file
  * svg
  *
+ * See _svg--embed.twig and _svg-sprite.twig for variables specific to those
+ * variants.
+ *
  * Available variables for svg:
  *  - svg.variant [string]: Indicates which svg pattern to use.
+ *  - svg.fillCurrent [bool]: Use the tailwind's fill-current utility class.
+ *  - svg.strokeCurrent [bool]: Use the tailwind's stroke-current utility class.
+ *  - svg_classes [array<string>]: Array of string classes to add
+ *      Example: ['text-red-500', 'w-10']
  */
 #}
 
-{% set svg_variant = svg.variant|default('embed') %}
+{# Default SVG settings, overridden by incoming svg object #}
+{% set svg = {
+  fillCurrent: true,
+  variant: 'embed',
+} | merge(svg) %}
 
-{# Classes Merge Handled by Variant Files #}
-{% set svg_classes = svg_classes|default([])|merge([
+{# Default SVG classes, overridden by incoming array #}
+{% set svg_classes = [
   'svg',
-  svg_variant,
-]) %}
+  'svg--' ~ svg.variant,
```

Comment:
With purgeCss do we need to update this to be full classes. 
`
```suggestion
{% set variants = {
    embed: ['svg--embed'],
    sprite: ['svg--sprite'],
} %}

{% set svg_classes = [
  'svg',
  svg.fillCurrent ? 'fill-current' : '',
  svg.strokeCurrent ? 'stroke-current' : '',
] 
  | merge(attribute(variants, svg.variant))
  | merge(svg_classes|default([]))
  | filter(c => c is not empty) | join(' ') %}

```

Reply:
ok, that suggestion was for not just replacing that one line. but lines 25-30

Reply:
@madhaze I don't think that will be completely necessary now that Tailwind implements PurgeCSS natively. It only purges itself.

Reply:
so we are only purgecssing tailwind classes?

Reply:
Correct! It's very cool!